runtime/src/lib.rs

```diff
@@ -596,6 +596,25 @@ impl pallet_preimage::Config for Runtime {
 	type ByteDeposit = PreimageByteDeposit;
 }
 
+parameter_types! {
+	pub const MaxResolvers: u32 = 10;
+	pub const MaxTags: u32 = 10;
+	pub const MaxEntries: u32 = 100;
+	pub const MaxServiceEndpoints: u32 = 10;
+	pub const SyloStringLimit: u32 = 500;
```

Comment:
could reduce space for these vars:
```suggestion
	pub const MaxResolvers: u8 = 10;
	pub const MaxTags: u8 = 10;
	pub const MaxEntries: u8 = 100;
	pub const MaxServiceEndpoints: u8 = 10;
	pub const SyloStringLimit: u16 = 500;
```